src/threads/table.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { dynamodb } from '../dynamodb';
+import { deleteAll, getAll, loadFixture } from '../testutils/db';
+import { ForumTable } from './table';
+
+describe('Forum table', () => {
+  const forumTable = new ForumTable(dynamodb);
+  const consoleErrorStub = jest.spyOn(global.console, 'error');
+  consoleErrorStub.mockImplementation(() => {});
+  beforeEach(async () => {
+    await deleteAll();
+  });
+
+  describe('getThreadEvents()', () => {
+    const participantId = 'participantId';
+    const itemId = 'itemId';
+    const pk = `THREAD#${participantId}#${itemId}`;
+    // @ts-ignore
+    const queryStub = jest.spyOn(forumTable.db, 'query');
+
+    it('should omit wrong entries', async () => {
+      await loadFixture([{ pk, time: 1, type: 'unknown_type' }]);
+      await expect(forumTable.getThreadEvents(participantId, itemId)).resolves.toEqual([]);
```

Comment:
If you want to demonstrate it only ignores the unexpected entries, you need at least to have valid entries to show they are still returned.